src/frontend/src/pages/MainPage/pages/homePage/components/McpServerTab.tsx

```diff
@@ -88,7 +215,7 @@ const McpServerTab = ({ folderName }: { folderName: string }) => {
   const MCP_SERVER_DEPLOY_TUTORIAL_LINK =
     "https://docs.langflow.org/mcp-server#deploy-your-server-externally";
 
-  const copyToClipboard = () => {
+  const copyToClipboard = useCallback(() => {
```

Comment:
No need for useCallback here the function is very light already and `MCP_SERVER_JSON` will be recreated on every render anyways causing the callback to update